Port the Termux KCM exclusions to Plasma 6.8.0.
Skip the Polkit-only date and time KCM when Polkit Qt is unavailable.
Disable mouse and touchpad KCMs without the KWin input-device D-Bus.
---
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5117af1..2c8269e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -91 +91 @@ find_package(PlasmaActivitiesStats ${PROJECT_DEP_VERSION} REQUIRED)
-find_package(PolkitQt6-1 REQUIRED)
+find_package(PolkitQt6-1 QUIET)
diff --git a/kcms/CMakeLists.txt b/kcms/CMakeLists.txt
index 9db9c83..84c1870 100644
--- a/kcms/CMakeLists.txt
+++ b/kcms/CMakeLists.txt
@@ -3 +3 @@ remove_definitions(-DQT_NO_CAST_FROM_ASCII -DQT_STRICT_ITERATORS -DQT_NO_CAST_FR
-add_subdirectory(libkwindevices)
+#add_subdirectory(libkwindevices)
@@ -6 +6,3 @@ add_subdirectory( access )
-add_subdirectory( dateandtime )
+if(PolkitQt6-1_FOUND)
+    add_subdirectory(dateandtime)
+endif()
@@ -24 +26 @@ add_subdirectory(qtquicksettings)
-add_subdirectory(touchscreen)
+#add_subdirectory(touchscreen)
@@ -32 +34 @@ add_subdirectory(solid_actions)
-add_subdirectory(mouse)
+#add_subdirectory(mouse)
@@ -34 +36 @@ add_subdirectory(mouse)
-add_subdirectory(touchpad)
+#add_subdirectory(touchpad)
@@ -37 +39 @@ if(BUILD_KCM_TABLET)
-    add_subdirectory(tablet)
+#    add_subdirectory(tablet)
